lighthouse/classes/plate_event.py

```diff
@@ -193,10 +199,10 @@ def errors(self) -> Dict[str, List[str]]:
         """
         error_message = {}
         for event_property_name in self.properties.keys():
-            if len(self.properties[event_property_name].errors) > 0:
+            if len(self.properties[event_property_name].errors) > 0:  # type: ignore
```

Comment:
We should know what type our error(s) are?

Reply:
Done